src/app/mini-games/sprint/utils/utils.ts

```diff
@@ -39,6 +39,30 @@ export function playRawSound(src: string) {
   audio.src = src;
   audio.play();
 }
+export function playRawSoundArr(arr: string[]) {
+  let sequence_position = 0;
```

Comment:
We usually use camelCase naming variable

Reply:
Done.